src/modules/cpu/kernel/vignette.hpp

```diff
@@ -0,0 +1,1377 @@
+#include "rppdefs.h"
```

Comment:
Add license on this file like https://github.com/ROCm/rpp/blob/master/src/modules/cpu/kernel/brightness.hpp

Reply:
Done